management-agent/management-tsa-common/src/main/java/com/terracotta/management/web/utils/TSAConfig.java

```diff
@@ -118,4 +119,13 @@ public static long getDefaultL1BridgeTimeout() {
       return DEFAULT_L1_BRIDGE_TIMEOUT;
     }
   }
+
+  public static long getDefaultL1BridgeConnectionTimeout() {
+    try {
+      String timeoutString = System.getProperty("com.terracotta.agent.defaultL1BridgeConnectionTimeout", "" + DEFAULT_L1_BRIDGE_CONNECTION_TIMEOUT);
```

Comment:
@GaryWKeim : FYI I have introduced a new system prop that can be set to change the default connection timeout (default to 7 sec). There was one for the read (call) timeout already.